ROMFS/px4fmu_common/init.d/rc.logging

```diff
@@ -3,13 +3,37 @@
 #
 # Standard startup script for logging
 #
+#
+# NOTE: environment variable references:
+#  If the dollar sign ('$') is followed by a left bracket ('{') then the
+#  variable name is terminated with the right bracket character ('}').
+#  Otherwise, the variable name goes to the end of the argument.
+#
+
+#
+# Declare local script variables and initialize default values.
+#
+set LOGGER_ARGS         ""
+set LOGGER_BUF          14
+
+#
+# Setup for board specific configurations.
+#
+if ver hwcmp PX4FMU_V5
+then
+	set LOGGER_BUF 64
+fi
+
+if param greater UAVCAN_ENABLE 1
```

Comment:
Can you move this into the `else` of `if ver hwcmp PX4FMU_V5`? On V5 we have enough RAM.

Reply:
Done.  Please let me know if I mis-interpreted your intention. Thanks!